Design.md

```diff
@@ -66,6 +66,10 @@ Investigate the use of Option types for error returns.
 
 By making the library header only, it becomes inured against variations in compiler flags. Whether it's a static or dynamic library becomes irrelevant, and there won't be impedance mismatches.
 
+### DUB Only Builds
+
+The current Phobos build process is convoluted and relies on old or niche tools. This makes the process of building Phobos tedious and discourages active community participation. DUB will be the only tool needed to build and test Phobos V3. Building and testing Phobos V3 should be as simple as issuing a `dub test` command.
```

Comment:
The problem with this is:

1. If the build is super simple (it is!), dub barely adds anything.
2. If the build isn't, dub can't actually do it.

Reply:
I am actually open to using another tool. I even rolled my own build tool for D during the 2018 DConf because DUB is ... not ideal. But at the moment it is the tool that the whole community knows/uses thus is follows that using it to build and test Phobos3 is a logical choice.